tests/10apidoc/01register.pl

```diff
@@ -126,7 +128,75 @@ sub matrix_register_user
    });
 }
 
-push @EXPORT, qw( local_user_fixture local_user_fixtures );
+push @EXPORT, qw( matrix_register_user_via_secret );
+
+sub matrix_register_user_via_secret
+{
+   my ( $http, $uid, %opts ) = @_;
+
+   my $password = $opts{password} // "an0th3r s3kr1t";
+   my $admin = $opts{admin} // 0;
```

Comment:
Purely stylistically - I tend to name boolean variables `$is_*`; so `$is_admin` in this case.